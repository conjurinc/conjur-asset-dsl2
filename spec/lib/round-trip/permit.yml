- !permit
  resource: !variable foobar
  privilege: read
  member: !group developers

- !permit
  resource: !variable baz
  privilege: [ read, execute ]
  member: !member
    role: !group developers

- !permit
  resource: !variable db_password
  privilege: read
  member:
    role: !group developers
    admin: true
