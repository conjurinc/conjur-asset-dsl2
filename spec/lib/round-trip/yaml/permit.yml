- !permit
  resource: !variable foobar
  privilege: read
  role: !group developers

- !permit
  role: !member
    role: !group developers
  resource: !variable baz
  privilege: [ read, execute ]

- !permit
  resource: !variable db_password
  privilege: read
  role: !member
    role: !group developers
    admin: true
