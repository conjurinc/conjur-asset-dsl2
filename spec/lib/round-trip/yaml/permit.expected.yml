---
- !permit
  privilege: read
  resource: !variable
    id: foobar
  role: !member
    role: !group
      id: developers
- !permit
  privilege:
  - read
  - execute
  resource: !variable
    id: baz
  role: !member
    role: !group
      id: developers
- !permit
  privilege: read
  resource: !variable
    id: db_password
  role: !member
    admin: true
    role: !group
      id: developers
